### ISSUE TITLE
Dokan reports driver installation fails when it actually succeeds but fails to start the service

### ISSUE THREAD

**Issue by Corillian** [dokan-dev/dokany#102]
`DokanCtl.exe` reports that installation of the driver fails when it actually succeeds but fails to start the driver service. This creates a problem as subsequent attempts to install the driver will fail because it's actually already installed however Dokan only reports this in debug output - not on the console. The primary problem is located in `DokanServiceInstall()` within mount.c in Dokan.dll.

Better output needs to be added so that the user actually knows what's going on instead of an arbitrary and misleading `Driver installation failed`.

**Comment by Liryna** [comment]
:+1: I agree
I will take a look for improving the informations

**Comment by kriswebdev** [comment]
I faced a similar issue after uninstalling Paragon ExtFS (not open source...) that relies on Dokany, probably version 0.6. System32 DLL and driver sys files couldn't be deleted ("this file is used by another program"). I wasn't able to properly uninstall the old dokany version throuh any known method (dokanctl /r a, Device Manager, pnputil, rundll32 DefaultUninstall, etc). That's until I found, trough Process Explorer (Find "dokan"), that explorer.exe was using dokannp.dll. So I killed explorer.exe and deleted dokannp.dll through cmd.

That makes me to my other point: **How to run different programs that are dependant of different versions of Dokany?** (ex. Paragon ExtFS with dokany driver 0.6 (that has its own mouting service) and encfs4win that I would like to upgrade to 0.8). I can put the DLL in my program directory but the driver file is the same (system32\driver\dokan.sys).
